src/util/math.hpp

```diff
@@ -63,4 +65,24 @@ namespace big::math
 		SHAPETEST::GET_SHAPE_TEST_RESULT(ray, &hit, &end_coords, &surface_normal, &hit_entity);
 		return end_coords;
 	}
+
+	// [0, max_value)
+	inline int rand(int max_value)
+	{
+		std::random_device seed;
+		std::mt19937 gen{seed()};
+		std::uniform_int_distribution<int> dist{0, max_value - 1};
+
+		return dist(gen);
+	}
+
+	// [min_value, max_value]
+	inline int rand(int min_value, int max_value)
+	{
+		std::random_device seed;
+		std::mt19937 gen{seed()};
```

Comment:
Seeding every time we call rand? Sounds like a bad idea.

Probably best to make `seed` and `gen` static variables, correct me if I'm wrong though.

Reply:
I can see why, though. I think every YimMenu user's spoofed host token is 41, even though we're seeding rand() already. That makes it easy to detect the menu in particular. That's not necessarily a problem, because who cares, but it's a common pattern I've noticed when observing users' host tokens.